presidio-analyzer/tests/test_uk_nino_recognizer.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from presidio_analyzer.predefined_recognizers import UkNinoRecognizer
+from tests.assertions import assert_result_within_score_range
+
+
+@pytest.fixture(scope="module")
+def recognizer():
+    return UkNinoRecognizer()
+
+
+@pytest.fixture(scope="module")
+def entities():
+    return ["UK_NINO"]
+
+
+@pytest.mark.parametrize(
+    "text, expected_len, expected_positions, expected_score_ranges",
+    [
```

Comment:
I would also modify some test cases to use context words

Reply:
Had `national insurance` in there and have added `nino`.

As the recognizer is being run outside of Analyzer Engine (which looks like the standard approach for all recognizer tests), the context works don't affect the score. So I've left the expected scores at 0.5.

Reply:
Sorry, for some reason I remembered context was possible in how we define the tests, approved